style.css

```diff
@@ -0,0 +1,110 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    
+}
+body{
+    background-color: #f4c531;
+
+}
+h1{
+    padding: 10px 10px 5px 10px;
+    font-size: 56px;
+    text-align: center;
+}
+#start{
+    display: flex;
+    margin: 0 auto;
+    padding: 5px;
+    font-size: 30px;
+    border-radius: 0.25em;
+    background-color: #ffffff;
+   
+    border: 0;
+}
+#stop{
+    margin: 20px 0 -25px 0;
+    font-size: 30px;
+    border-radius: 0.25em;
+    background-color: #f4c531;
+    padding: 5px;
+    border: 0;
+    text-align: center;
+}
+.wrapper{
+    box-sizing: border-box;
+    width: 39.0em;
+    margin: 50px;
+    padding: 2.0em 2.5em;
+    background-color: #ffffff;
+    position: absolute;
+    transform: translate(-50%,-50%);
+    left: 50%;
+    top:50%;
+    border-radius: 0.6em;
+    box-shadow: 0 0.9em 2.8em rgb(86, 66, 0 , 0.2);
+
+}
+.game-container{
+   flex-wrap: wrap;
+    width: 560px;
+    height: 430px;
+    display: flex;
+    gap: 10px;
+    transform-style: preserve-3d;
+    perspective: 500px;
+}
+.item{
+    position: relative;
+    width: 100px;
+    height: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 3em;
+    background-color: antiquewhite;
+    transform: rotateY(180deg);
+    transition: 0.25s;
+
+}
+.item, .boxOpen{
+    transform: rotateY(0deg);
+}
+.item::after{
+    content: '';
+    position: absolute;
+    inset: 0;
+    background: #f4c531;
+    transition: 0.25s;
+    transform: rotateY(0deg);
+    backface-visibility: hidden;
+}
+.boxOpen::after{
+    transform: rotateY(180deg);
+}
+
+@media (max-width:600px) {
```

Comment:
Please make sure to indent your code properly.

Reply:
i don't really understand here, but i formatted it with prettier